cmd/paths.go

```diff
@@ -108,14 +110,76 @@ func pathsAddCmd(ctx *config.Context) *cobra.Command {
 	return fileFlag(cmd)
 }
 
+func pathsEditCmd(ctx *config.Context) *cobra.Command {
```

Comment:
This `pathsEditCmd` is much more complicated than `pathsAddCmd`.
Can't you simplify it using `overWriteConfig` for example?

Reply:
Use `overWriteConfig`
https://github.com/hyperledger-labs/yui-relayer/pull/97/commits/ad6a4b7dae463f5f4c5eb39ad822e090647fd727